File: ios/Plugin/MapboxNavigationPlugin.swift
import Foundation

@objc public class MapboxNavigationPlugin: NSObject {
    @objc public func echo(_ value: String) -> String {
        print(value)
        return value
    }


    @objc public func navigateToDestination(_ latitude: Double, longitude: Double, userId: String){
      print("Latitude: ", latitude);
      print("Longitude: ", longitude);
      print("userId: ", userId);
    }
}

File: android/src/main/java/com/shovelapp/devfellow/mapboxnavi/MapboxNavigationPlugin.java
package com.shovelapp.devfellow.mapboxnavi;

import android.util.Log;

public class MapboxNavigationPlugin {

    public String echo(String value) {
        Log.i("Echo", value);
        return value;
    }


    public void navigateToDestination(double latitude, double longitude, String userId){
      Log.i("Latitude: ", latitude);
      Log.i("Longitude: ", longitude);
      Log.i("userId: ", userId);
    }
}
